dogu/doguVersionRegistry.go

```diff
@@ -40,72 +41,80 @@ func NewDoguVersionRegistry(configMapClient configMapClient) *doguVersionRegistr
 	}
 }
 
-func (vr *doguVersionRegistry) GetCurrent(ctx context.Context, name SimpleDoguName) (DoguVersion, error) {
+func (vr *doguVersionRegistry) GetCurrent(ctx context.Context, name dogu.SimpleName) (dogu.QualifiedVersion, error) {
 	descriptor, err := getDescriptorConfigMapForDogu(ctx, vr.configMapClient, name)
 	if err != nil {
-		return DoguVersion{}, err
+		return dogu.QualifiedVersion{}, err
 	}
 
 	currentDoguVersion, ok := descriptor.Data[currentVersionKey]
 	if !ok {
-		return DoguVersion{}, getDoguRegistryKeyNotFoundError(currentVersionKey, name)
+		return dogu.QualifiedVersion{}, getDoguRegistryKeyNotFoundError(currentVersionKey, name)
 	}
 
 	version, err := parseDoguVersion(currentDoguVersion, name)
 	if err != nil {
-		return DoguVersion{}, cloudoguerrors.NewGenericError(err)
+		return dogu.QualifiedVersion{}, cloudoguerrors.NewGenericError(err)
 	}
 
-	return DoguVersion{Name: name, Version: version}, nil
+	QualifiedName := dogu.QualifiedName{
+		SimpleName: name,
+	}
```

Comment:
Since this struct has no namespace it would not be without errors when `validate()` would be called on it. This feels weird to return such a struct. We should probably stay with the old DoguVersion-struct